tests/framework/icd/physical_device.h

```diff
@@ -74,6 +74,8 @@ struct PhysicalDevice {
 };
 
 struct PhysicalDeviceGroup {
+    PhysicalDeviceGroup() {}
+    PhysicalDeviceGroup(PhysicalDevice const& physical_device) { physical_device_handles.push_back(&physical_device); }
```

Comment:
Nit: Would be nice to also have a constructor that takes a vector of physical devices.
```c++
    PhysicalDeviceGroup(std::vector<PhysicalDevice*> const& physical_devices) {
         physical_device_handles.insert(physical_device_handles.end(), physical_devices.begin(), physical_device.end());
}

Reply:
I can add it, but since I initialize multiple groups from the same physical device vector, I don't think it will help my code at all.